components/installation-telemetry/cmd/send.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gitpod-io/gitpod/common-go/log"
+	"github.com/spf13/cobra"
+	"gopkg.in/segmentio/analytics-go.v3"
+)
+
+var segmentIOToken string
+
+var sendCmd = &cobra.Command{
+	Use:   "send",
+	Short: "Sends telemetry data",
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		// @todo(sje): replace with a database call to get status
+		canSendData := false
+		if !canSendData {
+			log.Info("installation-telemetry is not permitted to send - exiting")
+			return nil
+		}
+
+		if segmentIOToken == "" {
+			return fmt.Errorf("segmentIOToken build variable not set")
+		}
+
+		userId := os.Getenv("GITPOD_DOMAIN_HASH")
```

Comment:
We discussed the idea to give the admins the option to add a [pepper](https://en.wikipedia.org/wiki/Pepper_(cryptography)) as a secret that is read here for a re-hash to prevent that we are able to identify customers when we know the domain. Have you planned that for a follow-up PR or discarded the idea?

Reply:
This will be for a follow-up PR. However, this will be done in the actual `installation-telemetry` component as the secret will be optional and assigned to an envvar.

See #7538